types.bal

```diff
@@ -207,14 +193,23 @@ public type TimelineStepListResponse record {|
     int itemCount;
 |};
 
+public type TimelineSubstepPost record {|
+    string resultLocation;
```

Comment:
what is result location used for? Substeps have no need to record this information